melodies_monet/driver.py

```diff
@@ -1756,7 +1759,8 @@ def plotting(self):
                             if filter_criteria and 'altitude' in filter_criteria:
                                 vmin_y2, vmax_y2 = filter_criteria['altitude']['value']
                             elif filter_criteria is None:
-                                if 'altitude' in pairdf.columns:
+                                #if 'altitude' in pairdf.columns: # pairdf is dataset object, don't have columns
+                                if 'altitude' in list(pairdf.data_vars.keys()):
```

Comment:
This can be just
```if 'altitude' in paridf.keys()```

I also did the same change to the hotfix

Reply:
Also `if 'altitude' in pairdf` would work for both.

But for clarity I like
- `if var in df.columns` for pandas
- `if var in ds.variables` for xarray if the var might be a coordinate variable
  or `if var in ds.data_vars` if you know it's not a coord

Reply:
I revise it to if 'altitude' in pairdf.variables in the new commit, as the altitude might be a coordinate variable. I think this makes more sense.